composer.json

```diff
@@ -47,9 +47,9 @@
         "sonata-project/core-bundle": "^3.8",
         "symfony-cmf/resource": "^1.0",
         "symfony-cmf/resource-bundle": "^1.0",
-        "symfony-cmf/testing": "^2.0.1",
+        "symfony-cmf/testing": "^2.1",
```

Comment:
i guess, you should not update this dependency: let `2.0.*` be used with PHP5.6 and `2.1.*` - with PHP7.1 and higher

Reply:
mhh, ok, ill try it.